lib/suspenders/app_builder.rb

```diff
@@ -87,7 +87,7 @@ def enable_rack_deflater
     def setup_asset_host
       replace_in_file 'config/environments/production.rb',
         "# config.action_controller.asset_host = 'http://assets.example.com'",
-        'config.action_controller.asset_host = ENV.fetch("ASSET_HOST")'
+        'config.action_controller.asset_host = ENV.fetch("ASSET_HOST", ENV.fetch("HOST"))'
```

Comment:
Line is too long. [90/80]<br>Align the parameters of a method call if they span more than one line.